python/ray/scripts/scripts.py

```diff
@@ -550,12 +550,18 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
             # not provided.
             num_redis_shards = len(redis_shard_ports)
 
-        if address is not None:
+        address_env = os.environ.get("RAY_REDIS_ADDRESS")
+        underlying_address = \
+            address_env if address_env is not None else address
+        if underlying_address is not None:
             cli_logger.print(
-                "Will use value of `{}` as remote Redis server address(es). "
+                "Will use value of environment variable `{}`"
```

Comment:
How about `Will use {underlying_address} for external Redis server address(es)`, since this can be specified by both `RAY_REDIS_ADDRESS` env var and `--address` flag?

Reply:
done